scripts/ci/after_success.sh

```diff
@@ -13,8 +13,17 @@ if [ -z "$UPLOAD_URL" ]; then
     exit 1;
 fi
 
-cd sentinel && npm install --production && cd .. && \
-cd api && npm install --production && cd ..
+# Attempt to reduce size of api and sentinel artifacts
+(cd sentinel &&
```

Comment:
why the subshell/parens? so you don't need to manage the current working directory?  does it fail properly with the `set -e`?